src/ToggleBadge.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { theme as themeGet, space } from 'styled-system'
+import theme from './theme'
+
+const ToggleBadge = styled.div`
+  border-radius: 2px;
+  display: inline-block;
+  font-size: ${props => props.theme.fontSizes[0]}px;
```

Comment:
I think it might make sense to also allow this to be any fontSize passed in as a prop.